Makefile

```diff
@@ -3,11 +3,12 @@ include config.mk
 XAPIDOC=_build/install/default/xapi/doc
 JOBS = $(shell getconf _NPROCESSORS_ONLN)
 PROFILE=release
+XAPI_VERSION=$(shell git describe --always --dirty)
```

Comment:
I think that this would just override the variable, if you have specified it on the command line. Perhaps use `?=` instead of `=`.

Reply:
`make  -e ENV=var` works, but `ENV=var make` doesn't - i suspect `?=` will fix that